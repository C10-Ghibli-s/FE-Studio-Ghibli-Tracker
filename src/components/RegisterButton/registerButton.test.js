import React from 'react';
import TestRenderer from 'react-test-renderer';
import { CTA_RegisterButton } from '.';

const component = TestRenderer.create(
    <CTA_RegisterButton page='register'> Register Now! </CTA_RegisterButton>
).toJSON();


test('snapshot xd', () => {
    expect(component).toMatchSnapshot();
});
