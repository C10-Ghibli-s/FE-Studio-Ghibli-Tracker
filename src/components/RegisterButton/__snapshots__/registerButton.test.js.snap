// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot xd 1`] = `
<a
  className="registerNow"
  href="register"
>
   Register Now! 
</a>
`;
